service/controller/managementcluster/resource.go

```diff
@@ -143,6 +144,8 @@ func newResources(config resourcesConfig) ([]resource.Interface, error) {
 			K8sClient: config.K8sClient,
 			Logger:    config.Logger,
 
+			AlertmanagerEnabled: config.AlertmanagerEnabled && config.MimirEnabled,
```

Comment:
Funny indentation

Reply:
That's not an indentation :D I'm making sure this is only on capi

Reply:
Ignore me, that's my screen being small :D